pkg/controller/directvolumemigration/task.go

```diff
@@ -425,6 +425,25 @@ func (t *Task) Run() error {
 			return liberr.Wrap(err)
 		}
 		if completed {
+			isEgressNetworkSetup, err := hasAllRsyncClientPodsTimedOut(t.getPVCNamespaceMap(), t.Client, t.Owner.Name)
```

Comment:
NIT: `isEgressNetworkSetup` is kind of confusing. Feels like its conveying something is "right" when its set to True.  while it's actually the opposite. I'd like `isNetworkBusted` or something similar :)

Reply:
@pranavgaikwad I took care of that in a follow-up PR, renamed this to `isStunnelTimeout` to more accurately reflect the cause of the error. I am merging this